packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -395,7 +396,13 @@ Please create a release using "shorebird release" and try again.
           platform: platform,
           hash: hash,
           canSideload: false,
+          onProgress: (progress) {
```

Comment:
unrelated to this PR but would be nice to consolidate these two since the code looks almost identical.

Reply:
Agreed

Reply:
(will do before this is out of a draft state)